R/render_debrief_doc.R

```diff
@@ -0,0 +1,17 @@
+#' Render debriefing doc
+#'
+#' Take the markdown file and render a word planning doc to be used in workshops
+#'
+#' @param debrief_doc_info information about the workshop to be inserted into the debriefing document
```

Comment:
Here too I would check the name of the parameters and see if you can make them easier on the user. Simply `info` could suffice; though write in the documentation what function should be used to generate this object.